Support/PowershellSyntax.tmLanguage

```diff
@@ -324,13 +324,19 @@
 			<string>keyword.operator.other.powershell</string>
 		</dict>
 		<dict>
-			<key>comment</key>
-			<string>This is very imprecise, is there a syntax for 'must come after...' </string>
 			<key>match</key>
-			<string>(?&lt;!\s|^)\.\.(?=\d|\(|\$)</string>
+			<string>\.\.</string>
```

Comment:
I think a simple treatment of the Range Operator might be best. Whenever there is ```..```, the parser sees it as the range operator regardless of any issues before and after.